database-service/src/main/java/sjchat/daos/UserDaoImpl.java

```diff
@@ -0,0 +1,27 @@
+package sjchat.daos;
+
+import sjchat.entities.UserEntity;
+
+import javax.persistence.Query;
+import java.util.List;
+
+
+/**
+ * Created by Johan Vikström on 2017-04-01.
+ */
+public class UserDaoImpl extends DaoImpl implements UserDao{
+    public UserEntity findByUsername(String username){
+        Query q = em.createQuery("select u from UserEntity where u.username = :username", UserEntity.class);
+        q.setParameter("username", username);
+        List<UserEntity> users = q.getResultList();
+        if(users == null || users.isEmpty()){
+            return null;
+        }else{
```

Comment:
Format code according to Googles Java code specification. This applies for other files as well.